megaqc/rest_api/schemas.py

```diff
@@ -416,9 +416,9 @@ class FilterObjectSchema(BaseSchema):
     """
     A single filter object, e.g.
     {
-        'type': 'daterange',
-        'value': [],
-        'cmp': 'in'
+    'type': 'daterange',
```

Comment:
Sphinx complained when this was intended.
Looks worse now in code, but no warnings when generating the docs. But it's very minor, so whatever.